python/lsst/ts/audiotrigger/read_serial_temp_scanner.py

```diff
@@ -190,19 +192,19 @@ def config(self):
         # fan sensor
         self.fan_sensor = "Ambient"
         self.fan_gpio = 4
-        self.fan_turn_on_temp = 25
+        self.fan_turn_on_temp = 24
```

Comment:
Line 193 `self.fan_sensor = "Ambient"` that attribute is not used anywhere. Please remove it.

Reply:
Line 198 `self.configured = True` that attribute is not used anywhere. Please remove.